awx/main/models/workflow.py

```diff
@@ -184,6 +184,9 @@ class WorkflowJobNode(WorkflowNodeBase):
         default={},
         editable=False,
     )
+    do_not_run = models.BooleanField(
```

Comment:
I think it's worth having a description on this field at least to outline that it is part of the workflow execution resolution graph and the the information here should outline its meaning on said graph.

Reply:
I think this conversation is resolved by the `help_text` that @chrismeyersfsu just added?